README.md

```diff
@@ -138,6 +140,7 @@ simple page informing you that the bot succeeded/failed creating your hook.
 Assuming it succeeded, you should see a generic message appear in the channel
 you activated it in.
 
+It is recommended to set a `webhook_secret` in the `[github]` configuration section, and set it for the webhook at GitHub.com. This is only needed if `webhook_secret` was not set in the configuration prior to enabling the webhook.
```

Comment:
And even though I re-reviewed this just now, I _still_ missed this overly long line. Ugh.

```suggestion
It is recommended to set a `webhook_secret` in the `[github]` configuration
section, and set it for the webhook at GitHub.com. This is only needed if
`webhook_secret` was not set in the configuration prior to enabling the webhook.
```